app/src/main/java/de/danoeh/antennapod/fragment/preferences/ImportExportPreferencesFragment.java

```diff
@@ -181,70 +181,67 @@ private void exportDatabase() {
     }
 
     private void importDatabase() {
-        // setup the alert builder
-        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
-        builder.setTitle(R.string.database_import_label);
-        builder.setMessage(R.string.database_import_warning);
-
-        // add a button
-        builder.setNegativeButton(R.string.no, null);
-        builder.setPositiveButton(R.string.confirm_label, (dialog, which) -> {
+        new AlertDialog.Builder(getActivity())
```

Comment:
In this case, I find it easier to read the original version. It needs less indentation and shows more clearly what belongs together.